stressberry/cli.py

```diff
@@ -40,11 +40,7 @@ def _get_parser_run():
         help="name the data set (default: 'stressberry data')",
     )
     parser.add_argument(
-        "-t",
-        "--temperature-file",
-        type=str,
-        default="/sys/class/thermal/thermal_zone0/temp",
-        help="temperature file (default: /sys/class/thermal/thermal_zone0/temp)",
+        "-t", "--temperature-file", type=str, default=None, help="temperature file."
```

Comment:
Let's add the location of standard system file to the help text here. (`/sys/class/thermal/thermal_zone0/temp`) and add `(default: use vcgenmod)`.